modules/efk-logging-fluentd-external.adoc

```diff
@@ -77,29 +87,19 @@ not a path on your host system.
 ====
 +
 When configuring the external aggregator, it must be able to accept messages
-securely from Fluentd.
-
-. If the external aggregator is another Fluentd server, it must have the
-`fluent-plugin-secure-forward` plug-in installed and make use of the input
-plug-in it provides:
+securely from Fluentd and must have the `fluent-plugin-secure-forward` plug-in 
+installed and make use of the input plug-in it provides:
 +
 ----
 <source>
-  @type secure_forward
-
-  self_hostname ${HOSTNAME}
-  bind 0.0.0.0
-  port 24284
-
-  shared_key thisisasharedkey
-
-  secure yes
-  cert_path        /path/for/certificate/cert.pem
-  private_key_path /path/for/certificate/key.pem
-  private_key_passphrase secret_foo_bar_baz
+@type forward
+port 24224
+bind 0.0.0.0
 </source>
 ----
 
 You can find further explanation of how to set up the
-`fluent-plugin-secure-forward` plug-in in the
-link:https://github.com/tagomoris/fluent-plugin-secure-forward[`fluent-plugin-secure-forward` repository].
+`fluent-plugin-secure-forward` plug-in in the Fluentd documentation for 
+link:https://docs.fluentd.org/v1.0/articles/in_forward[Fluentd versions 1.0 or higher]
```

Comment:
Need to have a link for `out_forward` here too - https://docs.fluentd.org/v1.0/articles/out_forward - for Fluentd 1.0 or later.